src/core/Akka.Remote.Tests/RemoteActorTelemetrySpecs.cs

```diff
@@ -0,0 +1,150 @@
+
+namespace Akka.Remote.Tests
+{
+    public class RemoteActorTelemetrySpecs : AkkaSpec
+    {
+        // create HOCON configuraiton that enables telemetry and Akka.Remote
+        private static readonly string Config = @"
+            akka {
+                actor {
+                    provider = ""Akka.Remote.RemoteActorRefProvider, Akka.Remote""
+                    telemetry.enabled = true
+                }
+                remote {
+                    log-remote-lifecycle-events = on
+                    dot-netty.tcp {
+                        port = 0
+                        hostname = localhost
+                    }
+                }
+            }";
+
+        public RemoteActorTelemetrySpecs(ITestOutputHelper outputHelper) : base(Config, outputHelper)
+        {
+
+        }
+
+        private class TelemetrySubscriber : ReceiveActor
+        {
+            // keep track of integer counters for each event type
+            private int _actorCreated;
+            private int _actorStopped;
+            private int _actorRestarted;
+
+            // create a message type that will send the current values of all counters
+            public sealed class GetTelemetry
+            {
+                public int ActorCreated { get; }
+                public int ActorStopped { get; }
+                public int ActorRestarted { get; }
+
+                public GetTelemetry(int actorCreated, int actorStopped, int actorRestarted)
+                {
+                    ActorCreated = actorCreated;
+                    ActorStopped = actorStopped;
+                    ActorRestarted = actorRestarted;
+                }
+            }
+
+            public class GetTelemetryRequest
+            {
+                // make singleton
+                public static readonly GetTelemetryRequest Instance = new GetTelemetryRequest();
+
+                private GetTelemetryRequest()
+                {
+                }
+            }
+
+            public TelemetrySubscriber()
+            {
+                // Receive each type of IActorTelemetryEvent
+                Receive<ActorStarted>(e => { _actorCreated++; });
+                Receive<ActorStopped>(e => { _actorStopped++; });
+                Receive<ActorRestarted>(e => { _actorRestarted++; });
+                // receive a request for current counter values and return a GetTelemetry result
+                Receive<GetTelemetryRequest>(e =>
+                    Sender.Tell(new GetTelemetry(_actorCreated, _actorStopped, _actorRestarted)));
+            }
+
+            protected override void PreStart()
+            {
+                Context.System.EventStream.Subscribe(Self, typeof(IActorTelemetryEvent));
+            }
+        }
+
+        // create a unit test where a second ActorSystem connects to Sys and receives an IActorRef from Sys and subscribes to Telemetry events
+        [Fact]
+        public async Task RemoteActorRefs_should_not_produce_telemetry()
```

Comment:
Any `IActorRef` without an `ActorCell` will not produce metrics (by design) - so `FutureActorRef`s from `Ask<T>` and `RemoteActorRef`s don't influence these totals.